packages/commercetools/composables/src/helpers/internals/enhanceProduct.ts

```diff
@@ -37,7 +37,8 @@ const enhanceProduct = (productResponse: ApolloQueryResult<ProductData>, context
         _master: current.masterVariant.id === variant.id,
         _description: current.description,
         _categoriesRef: current.categoriesRef.map((cr) => cr.id),
-        _rating: (product as any).reviewRatingStatistics
+        _rating: (product as any).reviewRatingStatistics,
+        _type: product.productType?.name
```

Comment:
Maybe there's no use creating that key if product.productType doesn't exist? What about `...(product.productType && { _productType: product.productType })`? You'd also get the whole object if desired, not just the name prop.

Reply:
yeah, sounds better. thank you!